### PR TITLE
add myst syscalls to strace groups

### PR DESCRIPTION
**Pull request by salsal97**

### CODE REVIEW

File: utils/syscall.c [hunk 1]
```diff
@@ -583,7 +584,14 @@ static int _process_syscall_group[] = {SYS_clone,
                                        SYS_tgkill,
                                        SYS_waitid,
                                        SYS_rt_tgsigqueueinfo,
-                                       SYS_execveat};
+                                       SYS_execveat,
+                                       SYS_myst_get_process_thread_stack,
+                                       SYS_myst_get_exec_stack_option,
+                                       SYS_myst_pre_launch_hook,
```
Comment (vtikoo) [inline utils/syscall.c:590]: Consider moving this one to `myst` group as we discussed.

Comment (salsal97) [reply utils/syscall.c]: Added this!

**Review by vtikoo — APPROVED** [review]
Looks good!

**Comment by salsal97** [comment]
This PR fixes https://github.com/deislabs/mystikos/issues/1441

**Pull request merged**